### PR TITLE
client: do not override analyzer list from config file

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

File: cmd/client/client.go [hunk 1]
```diff
@@ -42,7 +42,9 @@ var ClientCmd = &cobra.Command{
 		if analyzerAddr != "" {
 			config.Set("analyzers", analyzerAddr)
 		} else {
-			config.SetDefault("analyzers", "localhost:8082")
+			if len(config.GetStringSlice("analyzers")) == 0 {
```
Comment (nplanel) [inline cmd/client/client.go]: why/how SetDefault override the current value of analyzers
Shall SetDefault("analyzers") rather be in config/config.go ?

Comment (lebauce) [reply cmd/client/client.go]: It's probably a bug in Cobra. When doing `config.SetDefault("analyzers", []string{"localhost:8082"})`, it seems to work correctly. I updated the PR

**Comment by masco-enovance** [comment]
run skydive-selenium-tests

**Comment by lebauce** [comment]
run skydive-selenium-tests

**Pull request merged**